data/json/construction.json

```
@@ -3895,7 +3895,7 @@
     "description": "Build Tempered Window",
     "//": "Step 2: Build curtain for tempered window",
     "category": "WINDOWS",
-    "required_skills": [ [ "fabrication", 9 ] ],
+    "required_skills": [ [ "fabrication", 4 ] ],
```

```suggestion
    "required_skills": [ [ "fabrication", 2 ] ],
```
Very complicated curtains!